### ISSUE TITLE
gh-contributors list for PDF-AAM is incorrect

### ISSUE THREAD

**Issue by cookiecrook** [w3c/pdf-aam#2]
https://w3c.github.io/pdf-aam/#acknowledgements

Currently lists Daniel and Philippe, but not James, Paul, or Zak... I'm not immediately sure where this list is coming from.

**Comment by jnurthen** [comment]
It comes from commits to the GH repository. 
I have an outstanding task to fix this for ARIA https://github.com/w3c/aria/issues/2238 so it will only look at the specific file in the ARIA repository rather than all files as it is now in ARIA - at the moment the PDF-AAM one is coming from the pdf-aam repo which only has issues in it hence it is incorrect.

Just to note though that gh-contributors will prune editors from that list so Paul and Zak will not be there (just as I am not in the list for the ARIA spec)

**Comment by daniel-montalvo** [comment]
This is one of the Respec functionalities that does not play well with monorepos. Logic is in the [contrib.js](https//github.com/speced/respec/blob/main/src/core/contrib.js) file

Even when this works properly for other specs, I am not sure what the value of this is really once we have automated the list of participants. With that, everyone will be listed there. The only case where this would add value is if we have a case where there's someone who has actively contributed through GitHub via PRs and is not an active participant, which is rare.

**Comment by Alexectramagneta** [comment]
keep up the great work guys

**Comment by zakkinsey** [comment]
@daniel-montalvo can we close this issue now?

**Comment by daniel-montalvo** [comment]
Yes. Respec currently filters out spec editors so I think we are good for now.
